maxim/logger/smolagents/client.py

```
@@ -0,0 +1,539 @@
+    return logging.DEBUG if debug else logging.WARNING
+
+
+class MaximEvalConfig:
+    """Maxim eval configuration for Smolagents."""
+    
+    def __init__(self):
+        self.evaluators = []
+        self.additional_variables = []
+
+
+def extract_smolagents_messages_and_params(agent, task, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `extract_smolagents_messages_and_params`

(ANN201)